examples/cryptocurrency-advanced/README.md

```diff
@@ -117,6 +118,12 @@ exonum-cryptocurrency-advanced run --node-config example/3/node.toml --db-path e
 exonum-cryptocurrency-advanced run --node-config example/4/node.toml --db-path example/4/db --public-api-address 0.0.0.0:8203 --master-key-pass pass
 ```
 
+Deploy cryptocurrency-advanced service:
+
+```sh
+python3 -m exonum_launcher -i ../cryptocurrency_advanced.yaml
```

Comment:
I suppose you need to describe the process of `exonum_launcher` module installing.

Reply:
There is a link to exonum-launcher above in the "Be sure you installed necessary packages:" section.